### PR TITLE
Add a terms of service

### PR DESCRIPTION
**Pull request by davidfischer**
- Link it from the privacy policy
- Link it from the footer

Once this is merged, we should link to it from our insertion order template.

### CODE REVIEW

File: content/pages/terms-of-service.md [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+Please read this Terms of Service agreement carefully before accessing or using EthicalAds.
+Because it is such an important contract between us and our users,
+we have tried to make it as clear as possible.
+For your convenience, we have presented these terms in a short non-binding summary
+followed by the full legal terms.
+
+If you are not a publisher or advertiser on our network,
+these terms generally do not apply to you.
+If you're just concerned about how we handle data from users
+who view ads on our network, see our [privacy policy]({filename}../pages/privacy-policy.md).
```
Comment (ericholscher) [inline content/pages/terms-of-service.md:17]: 👍

**Review by ericholscher — APPROVED** [review]
Looks good. Thanks for handling this ❇️

**Comment by ericholscher** [comment]
Merging so I can send it over to our advertiser asking for it.

**Comment by davidfischer** [comment]
I added the terms to our IO template

**Pull request merged**